src/service-worker.js

```diff
@@ -0,0 +1,29 @@
+var cacheName = 'invoizr'
```

Comment:
cacheName shouldn't be generated? otherwise, it would be caching the old assets each time we create a new build, instead of cache the new ones.

Maybe [this webpack plugin](https://github.com/goldhand/sw-precache-webpack-plugin) can help us.